apidaze/__init__.py

```diff
@@ -50,6 +51,8 @@ def __init__(
         self.cdr_handlers = Cdrhandlers(http=self.http)
         self.recordings = Recordings(http=self.http)
         self.misc = Miscellaneous(http=self.http)
+        self.media = None
+        self.sipusers = Sipusers(http=self.http)
```

Comment:
Shouldn't it be like the following?

```suggestion
        self.sip_users = Sipusers(http=self.http)
```

I, also, think the client/class names should follow the convention too such as `Sip_users`; which I see it should be retrospectively applied to `Externalscripts` and `Cdrhandlers` classes.


```suggestion
        self.sip_users = Sip_users(http=self.http)
```

Reply:
I would prefer to do this in a separate PR, as a refactoring task.